lib/data_update_scripts/20200518173504_update_public_reactions_count_from_positive_reactions_count.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class UpdatePublicReactionsCountFromPositiveReactionsCount
+    def run
+      ActiveRecord::Base.connection.execute("UPDATE articles SET public_reactions_count = positive_reactions_count")
+      ActiveRecord::Base.connection.execute("UPDATE articles SET previous_public_reactions_count = previous_positive_reactions_count")
+      ActiveRecord::Base.connection.execute("UPDATE comments SET public_reactions_count = positive_reactions_count")
```

Comment:
So this is only 3 transactions!!! I don't think this will fail or timeout even though we have a lot of rows to update here. This should all happen pretty quickly. Not an expert here though so feel free to comment if you feel otherwise!